anesthetic/samples.py

```diff
@@ -429,6 +433,10 @@ def __init__(self, *args, **kwargs):
             if logL_birth is not None:
                 self._compute_nlive(logL_birth)
 
+            for param in self.columns:
+                if param not in self.limits:
+                    self.limits[param] = (self[param].min(), self[param].max())
```

Comment:
Exact copy as for MCMCSamples does the trick to get limits for `logL`, `weight` and `nlive`.

It is a repetition of the same lines, though. Would you prefer if this was in its own utils function?

Reply:
Nevermind, moved into its own method. Much better this way.